Jenkinsfile

```diff
@@ -6,6 +6,7 @@ defaultEnv = [
 	] + params
 
 def k8sVersions = ["1.12", "1.13", "1.14", "1.15", "1.16"]
+def latestReleasedVersion = "1.16"

```

Comment:
why not just take the last index of k8sVersions?

Reply:
My reasoning was that we'll probably add versions like 1.17  to this list which they under active development and I thought it would help reduce noise to run these jobs for the latest stable/released k8s version.
Thoughts?

Reply:
sgtm -- ty